geojson_to_osmium_extracts.py

```diff
@@ -43,10 +38,15 @@ def main():
         num_batches = 0
 
         for feature in gj["features"]:
-            name = feature["properties"][args.name_key]
+            # The input contains both LADs and TAs. An area name might exist in
+            # both (like Portsmouth), so include the level in the namespace.
+            name = feature["properties"]["level"] + "_" + feature["properties"]["name"]
 
+            # TODO Fix upstream! All cases were checked right now and only using the first is OK.
```

Comment:
Devon and Torridge both have a tiny island; we just skip it. Isles of Scilly is the problem; it arbitrarily chooses one island:
![Screenshot from 2023-10-23 12-59-07](https://github.com/acteng/atip-data-prep/assets/1664407/a1d4cb04-ffed-4cfa-9bb0-7d741504700b)
Unless we're going to be collecting lots of data here, it's not an immediate problem. Splitting into a boundary per island is one option, or I can think about some harder fixes to route-snapper to handle this.